WcaOnRails/app/models/continent.rb

```diff
@@ -11,11 +11,11 @@ def self.country_ids(continent_id)
   end
 
   def name
-    I18n.t(recordName, scope: :continents)
+    I18n.t(read_attribute(:name), scope: :continents)
```

Comment:
I don't remember much here :(

Trying to do the same thoughts process here are some elements that probably made me chose the record name:

- "AfR", "ER", "NAR", etc... looks better to me as a key than "North America"
- Maybe I was afraid of how to deal with key containing space in YAML
- Maybe I did not know how to get the attribute "name" when inside a method "name" of the same model

I'm just trying to guess what I could have thought there, most likely the first point made me choose "recordName"!
It really doesn't remind me of any issue though.